src/main/java/com/ldtteam/structurize/client/gui/WindowScan.java

```diff
@@ -478,7 +478,7 @@ private void updateResources()
                 boolean handled = false;
                 for (final IPlacementHandler handler : PlacementHandlers.handlers)
                 {
-                    if (handler.canHandle(world, BlockPos.ZERO, blockState))
```

Comment:
This was weird, especially no need to load world at zero during compute resources step